_includes/install_instructions/python.html

```diff
@@ -42,107 +42,182 @@ <h3>Python</h3>
+              </li>
+            </ul>
+          </li>
+          <li>In the Miniforge Prompt window, run the following commands:
+            <ul>
+              <li>Type <code>cd Downloads</code> and press <kbd>Enter</kbd></li>
+              <li>Type <code>conda env create -f carpentries_environment.yml</code> and press <kbd>Enter</kbd></li>
+            </ul>
+          </li>
+          <li>The installation may take a few minutes depending on your internet connection speed. Once complete, you can close the Miniforge Prompt window.</li>
         </ol>
       </article>
       <article role="tabpanel" class="tab-pane" id="python-macos">
         <ol>
           <li>Open <a href="https://conda-forge.org/download/">https://conda-forge.org/download/</a> with your web browser.</li>
           <li>Download the appropriate Miniforge installer for macOS<br>
-	    (The following steps requires using the shell. If you aren't
-            comfortable doing the installation yourself
-            stop here and request help at the workshop.)
-	  </li>
+	          <em>The following steps requires using the shell. If you aren't comfortable doing the installation yourself stop here and request help at the workshop</em>.
+	      </li>
           <li>
-            Open a terminal window and navigate to the directory where
-            the executable is downloaded (e.g., <code>cd ~/Downloads</code>).
+            Open a terminal window and navigate to the directory where the executable is downloaded (e.g., <code>cd ~/Downloads</code>).
           </li>
           <li>
-            Type <pre>bash Miniforge3-</pre> and then press
-            <kbd>Tab</kbd> to autocomplete the full file name. The name of
-            file you just downloaded should appear.
+            Type <pre>bash Miniforge3-</pre> and then press <kbd>Tab</kbd> to autocomplete the full file name.
+            The name of file you just downloaded should appear.
+            Press <kbd>Enter</kbd> (or <kbd>Return</kbd> depending on your keyboard).
           </li>
           <li>
-            Press <kbd>Enter</kbd>
-            (or <kbd>Return</kbd> depending on your keyboard).
-            You will follow the text-only prompts.
-            To move through the text, press <kbd>Spacebar</kbd>.
-            Type <code>yes</code> and press enter to approve the license.
-            Press <kbd>Enter</kbd> (or <kbd>Return</kbd>)
-            to approve the default location
-            for the files.
-            Type <code>yes</code> and press
-            <kbd>Enter</kbd> (or <kbd>Return</kbd>)
-            to prepend Miniforge to your <code>PATH</code>
-            (this makes the Miniforge distribution the default Python).
+            Follow the text-only prompts in the terminal. To move through the text, press <kbd>Spacebar</kbd>.
+            <ul>
+              <li>Type <code>yes</code> and press enter to approve the license.</li>
```

Comment:
```suggestion
              <li>Type <code>yes</code> and press <kbd>Enter</kbd> (or <kbd>Return</kbd>) to approve the license.</li>
```

For consistency with the lines that follow.